### PR TITLE
fix: enabled OPA validation for preview resources 

### PR DESCRIPTION
**Pull request by olensmar**
and resources referred to by kustomizations - #1735 

This PR...

## Changes

-

## Fixes

-

## How to test it

-

## Screenshots

-

## Checklist

- [X] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/redux/services/validation.ts [hunk 1]
```diff
@@ -75,10 +72,6 @@ function isManagedByKustomize(resource: K8sResource): boolean {
     return true;
   }
 
-  if (resource.refs?.some(hasKustomizationReference)) {
```
Comment (olensmar) [inline src/redux/services/validation.ts:78]: this check excludes vanilla resources that are used in a kustomization so I removed it - couldn't come up with a scenario where those should be removed!?

**Review by topliceanurazvan — APPROVED** [review]

File: src/redux/services/validation.ts (hunk 1 above)
Comment (WitoDelnat) [reply src/redux/services/validation.ts:78]: It gives false positives. I wrote an example in #1695.

Comment (olensmar) [reply src/redux/services/validation.ts:78]: thanks for clarifying @WitoDelnat - I understand now although I think this scenario is much less common than just including valid resources in your kustomization setup (which seems to be the common case), so I suggest we remove this check for now. I'd rather have the UI show false positives in the uncommon scenario vs false negatives in the common one.

Going forward we we could improve the exclusion logic to exclude resources that are used by kustomizations that contain patches (would have to be recursive) - in which case I also think we should mark in the UI if/why a resource was excluded from validation so users don't incorrectly believe their resources are valid when they actually aren't.

thoughts @devcatalin ? @chlete ?

Comment (chlete) [reply src/redux/services/validation.ts:78]: I also agree with @olensmar with following the most common scenario and improve as we move forward.

**Pull request merged**